packages/backend/server/.env.example

```
@@ -1,15 +1,15 @@
-# DATABASE_URL="postgres://affine:affine@localhost:5432/affine"
-# REDIS_SERVER_HOST=localhost
-# COPILOT_FAL_API_KEY=YOUR_KEY
-# COPILOT_OPENAI_API_KEY=YOUR_KEY
-# COPILOT_PERPLEXITY_API_KEY=YOUR_KEY
+ DATABASE_URL="postgres://affine:affine@localhost:5432/affine"
```

dotenv-linter (4.0.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)